types/clamscan/.eslintrc.json

```diff
@@ -0,0 +1,5 @@
+{
+    "rules": {
+        "@definitelytyped/npm-naming": "off"
```

Comment:
why disable this rule? `clamscan` exists so this package has the correct name. And you didn't change any exports.